src/renderer/components/ft-playlist-selector/ft-playlist-selector.js

```diff
@@ -0,0 +1,75 @@
+import Vue from 'vue'
+import FtIconButton from '../ft-icon-button/ft-icon-button.vue'
+import { mapActions } from 'vuex'
+
+export default Vue.extend({
```

Comment:
`defineComponent` gives us auto-complete and is compatible with Vue 3.
```suggestion
import { defineComponent } from 'vue'
import FtIconButton from '../ft-icon-button/ft-icon-button.vue'
import { mapActions } from 'vuex'

export default defineComponent({
```